util/scrape.py

```diff
@@ -0,0 +1,75 @@
+from bs4 import BeautifulSoup
+import json
+import requests
+import re
+
+class Scraper:
+    """
+    Scrapes foosball rules from the official foosball website
+    and produces a rules.json file.
+    """
+    def parse_long(self):
+        """Parses the long rules list"""
+        self.setup_soup("http://www.foosball.com/learn/rules/ustsa/")
+        return self._dfs(self.soup.ol)
+
+    def parse_short(self):
+        """Parses the short rules list"""
+        self.setup_soup("http://www.foosball.com/learn/rules/onepage/")
+        return self._short()
+
+    def setup_soup(self, url):
+        """initializes the soup object given a url"""
+        html = requests.get(url).text
+        self.soup = BeautifulSoup(html, 'lxml')
+
+    def _dfs(self, node):
+        """
+        perform a dfs and make a list of lists representing a tree of
+        rules
+        """
+        if len(node.contents) <= 1:
+            return node.text.replace("\n", "").replace("\r", "")
+        else:
+            return [self._dfs(i) for i in node.findChildren()]
+
+    def _short(self):
+        """implmentation method for short html parse"""
```

Comment:
What is not consistently capitalized? The file name? The class name? Comments? The module name is lower case and the class name is capitalized. This coincides with PEP.